### PR TITLE
[14.0] dotfiles update needs manual intervention

### PR DESCRIPTION
**Pull request by etobella**
Dear maintainer,

After updating the dotfiles, `pre-commit run -a`
fails in a manner that cannot be resolved automatically.

Can you please have a look, fix and merge?

Thanks,

### CODE REVIEW

File: companyweb_payment_info/i18n/nl.po [hunk 1]
```diff
@@ -67,8 +67,8 @@ msgid ""
 msgstr ""
 "<h2>Companyweb upload</h2>Je staat op het punt je in te dienen <strong>%s</"
 "strong> openstaande factuur <br/>to Companyweb voor het bedrijf <strong>%s</"
-"strong>.<br/><strong>Het is de eerste keer dat u deze functie voor dit "
-"bedrijf gebruikt </strong><br/>Uw odoo-login : <strong>%s</strong> zal aan "
+"strong>.<br/>De vorige verzonden periode is <strong>%s</strong><br/>"
+"gc Uw odoo-login : <strong>%s</strong> zal aan "
```
Comment (huguesdk) [inline companyweb_payment_info/i18n/nl.po]: what is “gc” here?

**Review by huguesdk — COMMENTED** [review]
just something that feels like a mistake in a `.po` file, otherwise lgtm.

File: companyweb_payment_info/i18n/nl.po (hunk 1 above)
Comment (robinkeunen) [reply companyweb_payment_info/i18n/nl.po]: well spotted

**Pull request merged**